setup.cfg

```diff
@@ -2,6 +2,6 @@
 universal = 1
 
 [tool:pytest]
-addopts = -vs --tb=short --pep8 --flakes
+addopts = -vs --tb=short --pep8 --flakes --ignore docs/conf.py
```

Comment:
Not sure what I was doing here:  `addopts = -vs --tb=short --pep8 --flakes --ignore docs/conf.py`

Reply:
Update docs